shell/platform/windows/platform_handler.cc

```diff
@@ -405,8 +416,9 @@ void PlatformHandler::HandleMethodCall(
   if (method.compare(kExitApplicationMethod) == 0) {
```

Comment:
We cannot assume that the JSON is well-formed as that can cause crashes. Please use logic similar to `Clipboard.setData` to validate the arguments: https://github.com/flutter/engine/blob/686552f9444af6b169fbe522fc1987d99be88586/shell/platform/windows/platform_handler.cc#L429-L438